Sources/NIOSSL/SSLErrors.swift

```diff
@@ -106,6 +114,10 @@ public enum BoringSSLError: Error {
     case unknownError(NIOBoringSSLErrorStack)
     case invalidSNIName(NIOBoringSSLErrorStack)
     case failedToSetALPN(NIOBoringSSLErrorStack)
+
+    public static func failedToUsePrivateKey(_ errorStack: NIOBoringSSLErrorStack) -> FailedToUsePrivateKeyError {
```

Comment:
wasn't quite sure if that should go into `BoringSSLError` or `NIOSSLError`. Went with `BoringSSLError` because there were other errors holding an error stack :P